docs/mine/spacerace.md

```diff
@@ -165,6 +165,18 @@ If you would like to change the default gas fees to accelerate your messages, ed
   MaxWindowPoStGasFee = "50 FIL"
 ```
 
+## Why did my miner power suddenly drop?
+
+The following are some of the common reasons that can cause your power to be drop:
+
+1. Your PoSt messages are blocked by other messages which leads to not submitting PoSt for a deadline/window. To fix this, you can follow this [guide](https://github.com/filecoin-project/lotus/blob/master/documentation/en/mining.md#separate-address-for-windowpost-messages) and setup an separate address just for windowPoSt messages.
+
+2. Your sector data is corrupted and PoSt cannot be generated with bad underlying data. Unfortunately, we currently have no easy way to detect corrupted sectors, and the entire *runPost* fails even if only one sector is marked as faulty with the current proof APIs. However, the proofs team is working on getting our miners better APIs to mitigate this issue!
+
+3. For smaller miners, when all sectors are put into a single window, missing just one deadline/window wipes out all the power for 24 hours. 
```

Comment:
Can you include an explanation of when/why this resumes?

Reply:
Im actually not sure. @arajasek does this happen when  the miner does not have enough CPU to do all the proves?

Reply:
@mishmosh Power will be recovered if a valid WindowPoSt is submitted 24 hours later, when the missed deadline reopens.